parameters.yaml

```diff
@@ -6,3 +6,12 @@ parameters:
     required: true
     type: string
     description: Cluster ID
+
+  OrganizationIdPathParameter:
+    name: organization_id
+    in: path
+    required: true
+    type: string
+    description: |
+        An ID for the organization to be created is the only required parameter.
+        This ID can only contain alpha-numeric characters, hyphens, underscores, and dots.
```

Comment:
That might be what we talked about in 2015, but I believe this is what the real validator is for org ids: https://github.com/giantswarm/companyd/blob/96668b4c7d398e7d59cfdaa7600bb12d3cc8ddbd/service/validator.go#L12

Unless there is some other validator up the chain somewhere.